.travis.yml

```diff
@@ -4,6 +4,9 @@ language: cpp
 os:
   - linux
   - osx
+compiler:
+  - clang
+  - gcc
 osx_image: xcode8.3
```

Comment:
It seems like XCode 8.3 is being used as part of the name for linux builds as well. I still can't figure out why travis is doing so.